rplugin/python3/gdb/app.py

```diff
@@ -14,17 +14,22 @@ class App:
     def __init__(self, vim, backendStr, proxyCmd, clientCmd):
         self.vim = vim
 
+        # Prepare configuration: keymaps, hooks, parameters etc.
+        self.config = getConfig(vim)
+        self.defineSigns(self.config)
+
         # Create new tab for the debugging view and split horizontally
-        vim.command("tabnew | sp")
+        cmd = "tabnew | setlocal nowinfixwidth | setlocal nowinfixheight | exe \"normal \<c-w>o\" | "
+        if self.config["split_orientation"] == 'vertical':
```

Comment:
Probably it makes sense to have the Vim command in the configuration like `split_command`, which could have value either `vs` or `split` or whatever the user chooses.
We'd need to verify that there are exactly two windows after the command executed.